examples/official-site/robots.txt

```diff
@@ -0,0 +1,8 @@
+User-agent: *
+Disallow: /examples/authentification/  # Block the folder "authentification" to avoid blocking by password
+Disallow: /examples/hash_password.sql  # Block access to specific sensitive SQL files like this
+Disallow: /Dockerfile  # Block access to Dockerfile
```

Comment:
Why ? https://sql.datapage.app/Dockerfile

Reply:
I was told to limit the number of files that need to be crawled so I figure this one is not useful for Google